src/main/java/io/strimzi/kafka/bridge/mqtt/config/MqttConfig.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.config;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Represents configurations related to MQTT
+ * @see AbstractConfig
+ */
+public class MqttConfig extends AbstractConfig {
+
+    // Prefix for all the specific configuration parameters for MQTT in the properties file
+    public static final String MQTT_CONFIG_PREFIX = "mqtt.";
+
+    public static final String MQTT_HOST = MQTT_CONFIG_PREFIX + "host";
+
+    public static final String MQTT_PORT = MQTT_CONFIG_PREFIX + "port";
+
+    public static final String DEFAULT_MQTT_HOST = "localhost";
+
+    public static final int DEFAULT_MQTT_PORT = 1883;
+
+    public static String MAPPER_RULES_PATH;
```

Comment:
Is the plan to store the mapper rules in this variable? I don't think this variable is being used right now

Reply:
yes, we are not using, I forgot to remove it.